app/models/accounting/payment.rb

```diff
@@ -111,7 +111,7 @@ def expiration_date
 
       def format_data
         # Ensure the year is 4 digit representation
-        self.year = '20' + year.to_s[-2..-1].to_s
+        self.year = '20' + year.to_s[-2..-1].to_s unless year == -1
```

Comment:
Why would the year be -1?

Reply:
https://github.com/CKDev/accounting/blob/master/app/services/accounting/payment_service.rb#L20